examples/14-community-hosp/README.md

```diff
@@ -0,0 +1,99 @@
+# Community-hospital model
+
+In this model, we have three states:
+
+- Susceptible individuals (in the community),
+- Infected individuals (in the community), and
+- Infected hospitalized individuals.
+
+Susceptible individuals may become hospitalized or not (so they have two possible transitions), and infected individuals may recover or (if hospitalized) stay infected but be discharged (so they go back as infected but in the community).
```

Comment:
I think this needs to clarification that "When Susceptible individuals become Infected, they may also be come hospitalized (so they have two possible transitions)." I didn't understand this part immediately on the first read-through, at it became clearer in the code comments below.

Reply:
Good point. I think we need to add a mermaid diagram here. That will make things easier. I'm putting a pin to it so we can address it later.